### ISSUE TITLE
beta and gamma of batch normalization are not trained?

### ISSUE THREAD

**Issue by zhixy** [DrSleep/tensorflow-deeplab-resnet#67]
In train.py, 'all_trainable' is all variables except for whose name include beta or gamma, so they are the beta and gamma of batch normalization, right? If so, I wanna know why they are not trained?

**Comment by DrSleep** [comment]
https://github.com/DrSleep/tensorflow-deeplab-resnet/issues/66
